frontend/javascripts/viewer/model/sagas/volume/proofreading/proofread_saga.ts

```diff
@@ -1101,29 +1138,21 @@ function* handleProofreadMergeOrMinCut(action: Action) {
         },
       ),
     );
+    // Now the agglomerate Ids have changed again, thus, reload them.
+    yield* call(reloadMappingAndAggloIds);
 
     console.log("finished updating the mapping after a min-cut");
   }
 
   if (action.type === "PROOFREAD_MERGE") {
     // Remove the segment that doesn't exist anymore.
-    yield* put(removeSegmentAction(targetAgglomerateId, volumeTracingId));
+    // TODOM: in my test case this is wrong!
+    // TODOM: this breaks segment list syncing.! -> maybe removes mesh as well :thinking:
+    // yield* put(removeSegmentAction(targetAgglomerateId, volumeTracingId));
```

Comment:
There are edge cases where `targetAgglomerateId` when updated before this operation is wrong / outdated. Same for using the original `targetInfo.agglomerateId` information to remove the correct segment from the segment list. Therefore, I disabled this for now. This should be done correctly in live m4. 

But this is breaking my tests as removing the wrong segment auto removes the mesh with it -> accidentally deleting an auxiliary mesh which should instead still exist.